### PR TITLE
Add MI help-with-fees

### PR DESCRIPTION
**Pull request by damongreen123**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by timja — CHANGES_REQUESTED** [review]
this won't work without adding the environment specific files and given help-with-fees is already in a number of envs (aat,prod,preview,demo) adding this to the base requires that you add the env specific files in the same PR

**Review by timja — COMMENTED** [review]
Needs prod as well

**Comment by damongreen123** [comment]
> Needs prod as well

Cool, I'll get @cakeben  to add that when he has a spare. I dont have perms.

File: apps/help-with-fees/preview/base/kustomization.yaml [hunk 1]
```diff
@@ -2,4 +2,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../../../base
+  - ../../identity/identity.yaml
```
Comment (timja) [inline apps/help-with-fees/preview/base/kustomization.yaml:5]: this is missing the patch to add the aat identity file, see:
https://github.com/hmcts/cnp-flux-config/blob/3edbc3f79ea0c839b970a487147ffbeae66bf3ea/apps/rpe/preview/base/kustomization.yaml#L10

File: apps/help-with-fees/identity/aat.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
```
Comment (timja) [inline apps/help-with-fees/identity/aat.yaml:1]: All envs need the per env patch added, see:
https://github.com/hmcts/cnp-flux-config/blob/3edbc3f79ea0c839b970a487147ffbeae66bf3ea/apps/rpe/aat/base/kustomization.yaml#L8

**Review by timja — CHANGES_REQUESTED** [review]

**Review by timja — DISMISSED** [review]

**Comment by timja** [comment]
check the CI logs

File: apps/help-with-fees/identity/aat.yaml (hunk 2 above)
Comment (damongreen123) [reply apps/help-with-fees/identity/aat.yaml:1]: Added

**Review by timja — APPROVED** [review]

**Review by cakeben — APPROVED** [review]

**Pull request merged**